src/data/faq.json

```diff
@@ -1811,7 +1811,8 @@
                         "anchor": "mutation",
                         "active": false,
                         "textblock": [
-                            "<b>UPDATE</b><br/>To date, the risk assessment was improved and adapted on the 23.02.2021, on the 19.03.2021 and on the 16.04.2021. More information can be found in our blog:",
+                            "<b>UPDATE</b><br/>Information on the Corona-Warn-App and omicron can be found in this blog post: <a href='../blog/2022-01-11-cwa-omikron' target='_blank'>Wird die CWA an die Omikron-Variante angepasst?</a> (German only)",

```

Comment:
"omicron" => "Omicron"
It is a proper noun and should be capitalized.
Reference: 
https://www.who.int/news/item/26-11-2021-classification-of-omicron-(b.1.1.529)-sars-cov-2-variant-of-concern
"Based on the evidence presented indicative of a detrimental change in COVID-19 epidemiology, the TAG-VE has advised WHO that this variant should be designated as a VOC, and the WHO has designated B.1.1.529 as a VOC, named Omicron."

(German only) would look better with a full stop to end the sentence.

Reply:
I will take this into account, thanks.

Reply:
Done in https://github.com/corona-warn-app/cwa-website/commit/ac5b1b5a05053a386c3850a3a8bcbe248b6a935a.
Sorry for the default name of the commit, I used a workaround to make this changes as I'm not near my machine (which I provided this PR with) at the moment.